pandas/_libs/tslibs/resolution.pyx

```diff
@@ -399,245 +358,14 @@ cdef object month_position_check(fields, weekdays):
         return None
 
 
-cdef inline bint _is_multiple(int64_t us, int64_t mult):
+@cython.returns(bint)
```

Comment:
why don't you just type it and maybe cpdef?

Reply:
It isn't called from cython, so no use for cpdef (and cython won't allow `def bint ...`).  For that matter might as well just move it along with FrequencyInferer.